pkg/nfd-master/nfd-master.go

```diff
@@ -198,12 +198,10 @@ func newDefaultConfig() *NFDConfig {
 // Run NfdMaster server. The method returns in case of fatal errors or if Stop()
 // is called.
 func (m *nfdMaster) Run() error {
-	klog.Infof("Node Feature Discovery Master %s", version.Get())
+	klog.InfoS("Node Feature Discovery Master", "version", version.Get(), "nodeName", m.nodeName, "namespace", m.namespace)
 	if m.args.Instance != "" {
-		klog.Infof("Master instance: %q", m.args.Instance)
+		klog.InfoS("Master instance", "instance", m.args.Instance)
```

Comment:
do we need `"instance" `addition  here?

Reply:
Sorry, I didn't quite understand the comment 🤔 Do you mean removing the arg like

```golang
klog.InfoS("Master instance" m.args.Instance)
```

That's not possible because args must be in key-value pairs

Reply:
No, I I was thinking  `klog.InfoS("Master", "instance", m.args.Instance)` but this will result in "Master" instance="value" which looks a bit weird. I think then what you have is better.